public/data/prajwalborkar.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "Prajwal Borkar",
+    "type": "personal",
+    "bio": "Im a full stack developer specializing in building exceptional digital experiences. Currently focused on building responsive full stack web applications.I am also an open-source enthusiast.",
+    "avatar": "https://github.com/prajwalborkar.png",
+    "links": [
+      {
+        "name": "Checkout my GitHub",
+        "url": "https://github.com/prajwalborkar",
+        "icon": "github"
+      },
+      {
+        "name": "Connect me on Linkedln",
+        "url": "https://www.linkedin.com/in/prajwal-borkar/",
+        "icon": "linkedin"
+      }
+    ]
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```